src/cmd/launch.rs

```diff
@@ -22,3 +123,171 @@ impl LaunchStruct {
         LaunchStruct
     }
 }
+
+fn initialize_setting<DO: Downloader>(
+    launch: &command::Launch<DO>,
+) -> Result<(), Box<dyn std::error::Error>> {
+    // rootfsの種類などが記載された設定ファイルsetting.yamlを~/.injesh/containers/に作成する
+    match Path::new(&format!("{}/{}", launch.user().containers(), launch.name())).exists() {
+        true => return Err(Error::AlreadyExists)?,
+        false => {
+            fs::create_dir_all(format!(
+                "{}/{}/upper",
+                launch.user().containers(),
+                launch.name()
+            ))?;
+            let mut setting_file = fs::File::create(format!(
+                "{}/{}/setting.yaml",
+                launch.user().containers(),
+                launch.name()
+            ))?;
+            // TODO: 設定ファイルの形式を決めてない
+            setting_file.write_all(b"content of setting.yaml")?;
+        }
+    }
```

Comment:
重複している部分を解消

setting.yaml -> settings.yaml

```suggestion
) -> Result<(), Box<dyn std::error::Error>> {
    // rootfsの種類などが記載された設定ファイルsettings.yamlを~/.injesh/containers/に作成する
    let launch_container_path = format!("{}/{}", launch.user().containers(), launch.name());
    let launch_container_path = std::path::Path::new(&launch_container_path);
    if launch_container_path.exists() {
        Err(Error::AlreadyExists)?
    } else {
        fs::create_dir_all(launch_container_path.join("upper"))?;
        let mut setting_file = fs::File::create(launch_container_path.join("settings.yaml"))?;
        // TODO: 設定ファイルの形式を決める
        setting_file.write_all(b"content of settings.yaml")?;
    }
```